instrumentation/opencensus-shim/javaagent/src/main/java/io/opentelemetry/opencensusshim/OpencensusShimInstrumentationModule.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.Arrays;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class OpencensusShimInstrumentationModule extends InstrumentationModule {
+
+  public OpencensusShimInstrumentationModule() {
+    super("opencensus-shim", "io.opentelemetry.opencensusshim");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Arrays.asList(new OpenTelemetryCtxInstrumentation());
+  }
+
+  @Override
+  public boolean isHelperClass(String className) {
+    return className.equals("io.opentelemetry.opencensusshim.ContextExtractor");
+  }
```

Comment:
`isHelperClass` shouldn't be needed after updating the package name

Reply:
OH I think I'm piecing that together now -- I don't think I ever knew that was required to make that work.

I tried to move this as you suggested, but unfortunately, this one's a bit of a pickle: the `ContextExtractor` class relies on being in that package to access the package-private `io.opentelemetry.opencensusshim.OpenTelemetrySpanImpl` class. I can probably split up the bits into static pieces though? Any other suggestions on how to address that?